client/src/components/ListCover.jsx

```diff
@@ -1,59 +1,101 @@
 import {
+  Box,
   Card,
+  CardActionArea,
   CardContent,
   CardMedia,
   Grid,
   makeStyles,
+  Switch,
   Typography,
-} from "@material-ui/core"
-import React, { useContext } from "react"
-import sample from "../images/sample.jpg"
-import { userContext as context } from "../providers/UsersProvider"
+} from "@material-ui/core";
+import React, { useContext, useState } from "react";
+import { userContext as context } from "../providers/UsersProvider";
 
 const useStyles = makeStyles(() => ({
+  cardContainer: {
+    position: "relative",
+  },
   card: {
     width: 250,
     height: 350,
     backgroundColor: "white",
+    position: "relative",
   },
   image: {
-    height: 260,
+    height: 280,
   },
-}))
+}));
 
 const ListCover = ({ list }) => {
-  const classes = useStyles()
-  const userContext = useContext(context)
+  const classes = useStyles();
+  const userContext = useContext(context);
 
   const {
     updateIsListClicked,
+    axiosWithAuth,
+    openSnackbar,
     updateCurrentList,
     getListProducts,
-  } = userContext
+  } = userContext;
+
+  const { title, id, items, imageUrl, isPrivate } = list;
 
-  const { title, id, items, imageUrl } = list
+  const [toggle, setToggle] = useState(isPrivate);
```

Comment:
it's actually not good practice to duplicate a prop in state. The proper way is to just keep using `isPrivate` and call a function in the parent component to do the toggling

Reply:
oh, so it would be to have a function in the parent component that does a filter(to find the toggled list) and then updates the lists' state, right?